### ISSUE TITLE
Different Opening Questions

### ISSUE THREAD

**Issue by Glyciant** [Glyciant/PurplePlus#20]
Reddit Suggestion:

https://www.reddit.com/r/Twitch/comments/6xz6e5/the_wait_is_over_purple_is_now_here_important/dmkl24x/

**Comment by kot91** [comment]
I would agree with Crunchprank and I felt the same filling out those fields.

Different questions might be "In one sentence, describe your stream" as a replacement for "intro".

About you stays the same

and background could be changed to professional/hobbyist background as it pertains to what you're streaming or being a broadcaster in general. (CM, audio technician, esports, self taught artist, formally educated, etc)
